flow360/component/simulation/models/volume_models.py

```diff
@@ -194,65 +231,148 @@ def validate_consistent_array_length(cls, values):
 
 
 class ActuatorDisk(Flow360BaseModel):
-    """Same as Flow360Param ActuatorDisks.
+    """:class:`ActuatorDisk` class for setting up the inputs for an Actuator Disk.
+    Please refer to the :ref:`actuator disk knowledge base <knowledge_base_actuatorDisks>` for further information.
     Note that `center`, `axis_thrust`, `thickness` can be acquired from `entity` so they are not required anymore.
     """
 
-    entities: EntityList[Cylinder] = pd.Field(alias="volumes")
-    force_per_area: ForcePerArea = pd.Field()
-    name: Optional[str] = pd.Field(None)
+    entities: Optional[EntityList[Cylinder]] = pd.Field(
```

Comment:
Can you change entities definition here to be consistent with the latest one?

Reply:
All updated